dpdd.tex

```diff
@@ -1068,6 +1091,84 @@ \subsubsection{\ForcedSource Table}
 flags & bit[8] & bit & Various useful flags. \\
 \end{schema}
 
+\subsubsection{\texttt{ShearObject} Table}
+\label{sec:shearObjectTable}
+
+% ShearObject table is a conventions-translation and generalization of a schema
+% of the DES metadetect catalog schema from Matt Becker; differences include:
+%
+% x,y: dropped as redundant with ra/dec
+%
+% gauss_*: prefixes dropped as irrelevant in this context.
+%
+% gauss_T_ratio: dropped as redundant with T/TPSFMeta.  We might want to
+% restore it in the physical schema so we can index it separately.
+%
+% pgauss_*: prefixes replaced with "std[Flux]" to emphasize conceptual
+% relationship with Object's "stdColors".
+%
+% PSF moment measurements: I've assumed full (g1, g2, T) measurements on both
+% the metacalibration-modified PSFs and the originals, but dropped pgauss_psf_T
+% as too algorithmic-specific for the logical schema.
+%
+% I've expanded the pgauss_T field and its uncertainty into per-band
+% measurements, as the apparent cross-band average in the original schema also
+% seems too algorithm-specific, and I'd like to err on the side of predicting a
+% larger table than we might need.
+
+\begin{schema}{\texttt{ShearObject} Table}{Data Release Catalog \texttt{ShearObject} Table}{tbl:shearObjectTable}
+
+shearObjectId & uint64 & ~ & Unique object identifier. \\
+
+cellId & uint64 & ~ & Unique identifier for the coadd cell in which this object was detected and measured. \\
+
+metaStep & int8 & ~ & Enumeration value for which of the five \texttt{Metacalibration} modifications was applied to images prior to all measurements. \\
+
+radec & double[2] & degrees & Position of the object (centroid), measured jointly across all bands. \\
+
+maskFractionObj & float & ~ & Weighted fraction of this object's pixels that were masked, over all visit images input to the coadd. \\
+
+maskFractionCell & float & ~ & Weighted fraction the coadd cell's that were masked, over all visit images input to the coadd.\footnote{This and \texttt{nEpochCell} are the same for all rows with the same \texttt{cellId}, but are expected to be access per-object; they may or may not be normalized into a separate table in the physical schema.}. \\
+
+nEpochCell & int32[ugrizy] & ~ & Number of input visits that contributed to this coadd cell. \\
+
+g1 & float & ~ & Ellipticity from Gaussian-weighted moments, on-axis, measured consistently over central\footnote{Probably $riz$} bands. \\
```

Comment:
We don't know what the measurement will be yet.  Maybe just say "Ellipticity measured over multiple bands."  Not sure what you mean by "on-axis" or "over central"

Reply:
"on-axis" (wth "off-axes", below) was my attempt to very concisely explain `e1` vs `e2`, but it sounds like it didn't make any sense.  I'll just drop it for now, but let me know if you've got another way to word that.

"over central" is part of "over central bands", i.e. the bands we actually use for shear estimation vs. all bands.

I've updated all of this to be less specific and (hopefully) less confusing.